### PR TITLE
Added GitRemote version of the toolbar

### PR DESCRIPTION
**Pull request by ggqc007**
This is for issue #6

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -73,15 +73,15 @@
...
     </build>
-    <description>This plugins adds a toolbar for common GIT actions
+    <description>This plugins adds a toolbar for common Git/GitRemote actions
 &lt;p&gt;
 &lt;img src="https://raw.github.com/markiewb/nb-git-toolbar/master/doc/screenshot1.gif"/&gt;
 &lt;/p&gt;
 
 &lt;h2&gt;Updates&lt;/h2&gt;
-&lt;h3&gt;1.1.0:&lt;/h3&gt;
+&lt;h3&gt;1.2.0:&lt;/h3&gt;
```
Comment (markiewb) [inline pom.xml]: Please append your changes. The changelog has to contain even the previous changes.

File: src/main/resources/de/markiewb/netbeans/plugin/git/toolbar/layer.xml [hunk 2]
```diff
@@ -69,5 +69,73 @@
                 <attr name="position" intvalue="900"/>
             </file>
         </folder>
+
```
Comment (markiewb) [inline src/main/resources/de/markiewb/netbeans/plugin/git/toolbar/layer.xml]: Are you really sure that the toolbar is not visible, when I do not have a remote project?

Comment (ggqc007) [inline src/main/resources/de/markiewb/netbeans/plugin/git/toolbar/layer.xml]: Yes. The GitRemote bar will only show up if C/C++ is enabled in netbeans. If C/C++ is not enabled, the GitRemote Toolbar icons wont show up.

When C/C++ is enabled, one can choose to enable/disable the Git Toolbar or the GitRemote toolbar accordingly

**Comment by markiewb** [comment]
Thanks for the changes. I will have another look and try tomorrow!

With kind regards, markiewb

2017-06-06 20:30 GMT+02:00 GG <notifications@github.com>:

File: src/main/resources/de/markiewb/netbeans/plugin/git/toolbar/layer.xml (hunk 2 above)
Comment (ggqc007) [inline src/main/resources/de/markiewb/netbeans/plugin/git/toolbar/layer.xml]: Just to add some more precisions, the GitRemote actions becomes available in Netbeans as soon as  C/C++ is activated. You dont need to have an active (open) remote C/C++ project for the bar to show up although the icons will be gray.

At that point, we need to have a look at issue #7 and try to hide the bars completly when a different or no versionning system is used.

**Comment by markiewb** [comment]
I created a fork of the plugin. https://github.com/markiewb/nb-gitremote-toolbar
Your code is integrated there. Thank you!

**Pull request closed without merging**